src/hooks/UrqlUseSubscriptionWithHandler.re

```diff
@@ -1,38 +1,42 @@
+type handler('acc, 'resp, 'ret) =
```

Comment:
Now that we have our GADT `handler` here, I think we can get rid of `handler` and `parsedHandler` in `UrqlTypes.re`, correct?

Reply:
When removing the `handler` and `parsedHandler`, I noticed that `UrqlSubscription` uses them. I think it's better to leave them here for now and tackle it in a new PR focused on migrating `UrqlSubscription` to use this approach